modules/end-user-guide/pages/creating-a-workspace-from-a-code-sample.adoc

```diff
@@ -0,0 +1,7 @@
+[id="creating-a-workspace-from-a-code-sample"]
+
+:navtitle: Creating a workspace from a code sample
+:keywords: end-user-guide, creating-a-workspace-from-a-code-sample
+:page-aliases: .:creating-a-workspace-from-a-code-sample
```

Comment:
```suggestion
:page-aliases: .:creating-a-workspace-from-a-code-sample, .:creating-a-workspace-from-code-sample, creating-a-workspace-from-code-sample
```

Reply:
When renaming pages, add page-aliases to handle redirections from the older page name.